src/librustc/ty/relate.rs

```diff
@@ -407,13 +407,10 @@ pub fn super_relate_tys<'a, 'gcx, 'tcx, R>(relation: &mut R,
         }
 
         (&ty::TyDynamic(ref a_obj, ref a_region), &ty::TyDynamic(ref b_obj, ref b_region)) => {
-            let region_bound = relation.with_cause(Cause::ExistentialRegionBound,
-                                                       |relation| {
-                                                           relation.relate_with_variance(
-                                                               ty::Contravariant,
-                                                               a_region,
-                                                               b_region)
-                                                       })?;
+            let region_bound = relation.with_cause(
+                Cause::ExistentialRegionBound,
+                |relation| relation.relate_with_variance(ty::Contravariant, a_region, b_region),
+            )?;
```

Comment:
Is this just a drive-by formatting change? I'd prefer if it wasn't included, it feels confusing.

Reply:
Yes. If I recall correctly I was looking if I _needed_ to modify that file while exploring for a solution and left that small cleanup in because it was small enough, but I can revert it.